text/0000-rust-governance.md

```diff
@@ -0,0 +1,728 @@
+setting direction.
+
+A distinct problem in the other direction has also emerged recently: we've
+slowly been requiring RFCs for increasingly minor changes. While it's important
+that user-facing changes and commitments be vetted, the process has started to
+feel heavyweight (especially for newcomers), so a recalibration may be in order.
+
+We need a way to scale up the RFC process that:
+
+* Ensures each RFC is thoroughly reviewed by several people with
+  interest and expertise in the area, but with different perspectives
+  and concerns.
+
+* Ensures each RFC continues moving through the pipeline at a
+  reasonable pace.
+
+* Ensures that accepted RFCs are well-aligned with the values, goals,
+  and direction of the project, and with other RFCs (past, present,
+  and future).
+
+* Ensures that simple, uncontentious changes can be made quickly, without undue
+  process burden.
+
+### Scalability: areas of focus
+
+In addition, there are increasingly areas of important work that are
+only loosely connected with decisions in the core language or APIs:
+tooling, documentation, infrastructure, for example. These areas all
+need leadership, but it's not clear that they require the same degree
+of global coordination that more "core" areas do.
+
+These areas are only going to increase in number and importance, so we
+should remove obstacles holding them back.
+
+### Stakeholder involvement
+
+RFC shepherds are intended to reach out to "stakeholders" in an RFC,
+to solicit their feedback. But that is different from the stakeholders
+having a direct role in decision making.
+
+To the extent practical, we should include a diverse range of
+perspectives in both design and decision-making, and especially
+include people who are most directly affected by decisions: users.
+
+We have taken some steps in this direction by diversifying the core
+team itself, but (1) members of the core team by definition need to
+take a balanced, global view of things and (2) the core team should
+not grow too large. So some other way of including more stakeholders
+in decisions would be preferable.
+
+### Clarity and transparency
+
+Despite many steps toward increasing the clarity and openness of
+Rust's processes, there is still room for improvement:
+
+* The priorities and values set by the core team are not always
+  clearly communicated today. This in turn can make the RFC process
+  seem opaque, since RFCs move along at different speeds (or are even
+  closed as postponed) according to these priorities.
+
+  At a large scale, there should be more systematic communication
+  about high-level priorities. It should be clear whether a given RFC
+  topic would be considered in the near term, long term, or
+  never. Recent blog posts about the 1.0 release and stabilization
+  have made a big step in this direction. After 1.0, as part of the
+  regular release process, we'll want to find some regular cadence for
+  setting and communicating priorities.
```

Comment:
The process of setting priorities and strategic level direction has not been part of the RFC process, its been decided by the core team and only the results of the process have been stated publicly (not even the results sometimes). So it seems to be a process which is (somewhat) orthogonal to the RFC process and one where the level of openness has been sub-optimal.